lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/apriori/ExplanationResult.java

```diff
@@ -0,0 +1,62 @@
+package edu.stanford.futuredata.macrobase.analysis.summary.apriori;
+
+
+import java.util.Map;
+import java.util.StringJoiner;
+
+public class ExplanationResult {
+    private Map<String, String> matcher;
+    private double matchedOutlier, matchedCount;
+    private double totalOutlier, totalCount;
+
+    private double rateLift;
+    private double support;
+
+    public ExplanationResult(
+            Map<String, String> matcher,
+            double matchedOutlier,
+            double matchedCount,
+            double totalOutlier,
+            double totalCount
+    ) {
+        this.matcher = matcher;
+        this.matchedOutlier = matchedOutlier;
+        this.matchedCount = matchedCount;
+        this.totalOutlier = totalOutlier;
+        this.totalCount = totalCount;
+
+        this.rateLift = (matchedOutlier / matchedCount) / (totalOutlier / totalCount);
```

Comment:
This isn't the risk ratio: is the ratio of current outlier rate to global outlier rate. It's easier to compute, more well behaved, and plays better with the new pruning rules. 

Under risk ratio, if a "result" trivially matched a huge population including all the outliers it would have infinite risk ratio, masking more meaningful results.